jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java

```diff
@@ -156,39 +155,56 @@ public static Map<String, Object> shipData(Map<String, String> params) {
      */
     public static void doBackUpAndCleanUpInventory(String dataSource, String srcFolder, AWSCredentialProvider awsCredentialProvider) {
         LOGGER.info("Inside doBackUpAndCleanUpInventory Method:{}");
+        LOGGER.debug(dataSource);
+        LOGGER.debug(srcFolder);
 
         try {
             //Get the required params from System properties
             account = System.getProperty("base.account");
-            region  = System.getProperty("base.region");
-            s3Role  = System.getProperty("s3.role");
+            region = System.getProperty("base.region");
+            s3Role = System.getProperty("s3.role");
             s3Processed = System.getProperty("s3.processed");
-            s3Bucket    = System.getProperty("s3");
-            LOGGER.debug("base account: {}", account);
-            LOGGER.debug("base region: {}", region);
+            s3Data = System.getProperty("s3.data");
+            s3Bucket = System.getProperty("s3");
+            LOGGER.debug("printing account");
+            LOGGER.debug(account);
+            LOGGER.debug("printing region");
+            LOGGER.debug(region);
+            LOGGER.debug("printing s3Role");
+            LOGGER.debug(s3Role);
+            LOGGER.debug(srcFolder);
+            LOGGER.info("Trying to get Basic awsCredentials using IAM role");
             BasicSessionCredentials credentials = awsCredentialProvider.getCredentials(account, region, s3Role);
+            LOGGER.info("Printing credentials");
```

Comment:
Please remove creds from logging. No keys, tokens, etc should be in the log file or repo.